mne/tests/test_epochs.py

```diff
@@ -1750,7 +1751,9 @@ def test_saved_fname_no_splitting(
     dst_fpath = tmp_path / dst_fname
     split_1_fpath = tmp_path / split_1_fname
 
-    epochs.save(dst_fpath, split_naming=split_naming, verbose=True)
+    filenames = epochs.save(dst_fpath, split_naming=split_naming, verbose=True)
+    assert len(filenames) == 1
+    assert filenames[0] == Path(dst_fpath)
```

Comment:
Already a path so no need to cast, and can just check both length and content simultaneously
```suggestion
    assert filenames == [dst_fpath]
```